data/metadata.json

```diff
@@ -0,0 +1,5823 @@
+      "user_documentation": {
+        "specification": "docs/user/types/acknowledgments/acknowledgment/names/name-spec.en.md",
+        "usage": "docs/user/types/acknowledgments/acknowledgment/names/name-usage.en.md"
+      },
+      "label": "AcknowledgmentsItemsNamesItemsLabel",
+      "description": "AcknowledgmentsItemsNamesItemsDescription",
+      "type": "STRING",
+      "string_is_multiline": false,
+      "string_min_length": 1,
+      "examples": [
+        "Albert Einstein",
+        "Johann Sebastian Bach"
+      ],
+      "mandatory": {
+        "csaf_base": "",
+        "csaf_security_incident_response": "",
+        "csaf_informational_advisory": "",
+        "csaf_security_advisory": "",
+        "csaf_vex": ""
+      }
+    },
+    {
+      "path": "$.document.acknowledgments[].organization",
+      "user_documentation": {
+        "specification": "docs/user/types/acknowledgments/acknowledgment/organization-spec.en.md",
+        "usage": "docs/user/types/acknowledgments/acknowledgment/organization-usage.en.md"
+      },
+      "label": "AcknowledgmentsItemsOrganizationLabel",
+      "description": "AcknowledgmentsItemsOrganizationDescription",
+      "type": "STRING",
+      "string_is_multiline": false,
+      "string_min_length": 1,
+      "examples": [
+        "CISA",
+        "Google Project Zero",
+        "Talos"
+      ],
+      "mandatory": {
+        "csaf_base": "nice_to_know",
+        "csaf_security_incident_response": "nice_to_know",
+        "csaf_informational_advisory": "nice_to_know",
+        "csaf_security_advisory": "best_practice",
+        "csaf_vex": "nice_to_know"
+      }
+    },
+    {
+      "path": "$.document.acknowledgments[].summary",
+      "user_documentation": {
+        "specification": "docs/user/types/acknowledgments/acknowledgment/summary-spec.en.md",
+        "usage": "docs/user/types/acknowledgments/acknowledgment/summary-usage.en.md"
+      },
+      "label": "AcknowledgmentsItemsSummaryLabel",
+      "description": "AcknowledgmentsItemsSummaryDescription",
+      "type": "STRING",
+      "string_is_multiline": true,
+      "string_min_length": 1,
+      "examples": [
+        "First analysis of Coordinated Multi-Stream Attack (CMSA)"
+      ],
+      "mandatory": {
+        "csaf_base": "nice_to_know",
+        "csaf_security_incident_response": "nice_to_know",
+        "csaf_informational_advisory": "nice_to_know",
+        "csaf_security_advisory": "best_practice",
+        "csaf_vex": "nice_to_know"
+      }
+    },
+    {
+      "path": "$.document.acknowledgments[].urls",
+      "user_documentation": {
+        "specification": "docs/user/types/acknowledgments/acknowledgment/urls-spec.en.md",
+        "usage": "docs/user/types/acknowledgments/acknowledgment/urls-usage.en.md"
+      },
+      "label": "AcknowledgmentsItemsUrlsLabel",
+      "description": "AcknowledgmentsItemsUrlsDescription",
+      "type": "ARRAY",
+      "mandatory": {
```

Comment:
We should find a better name for `"mandatory"`...

Reply:
I agree. I changed it to `relevance_levels` according to the name of the spreadsheet we get the data from. Please let me know if you have another suggestion.